graylog2-web-interface/src/views/components/visualizations/heatmap/HeatmapVisualization.tsx

```diff
@@ -134,7 +120,7 @@ const _chartLayout = (heatmapData) => {
   return {
     yaxis: axisConfig,
     xaxis: axisConfig,
-    plot_bgcolor: hasContent ? BG_COLOR : 'transparent',
+    // plot_bgcolor: hasContent ? BG_COLOR : 'transparent',
```

Comment:
Why is this commented out?

Reply:
It must be deleted, we can't set the BG_COLOR anymore, since we do not have it, with the themes.